apps/docs/app/self-hosting/kubernetes/page.mdx

```diff
@@ -0,0 +1,274 @@
+kubectl get ingress -n formbricks
+```
+
+### **Access Formbricks**
+- If running locally with **Minikube**:
+  ```sh
+  minikube service my-formbricks -n formbricks
+  ```
+- If deployed on a **cloud cluster**, visit:
+  ```
+  https://formbricks.example.com
+  ```
+
+---
+
+## Upgrading Formbricks
+
+This section provides guidance on how to upgrade your Formbricks deployment using Helm, including examples of common upgrade scenarios.
+
+### Upgrade Process
+
+To upgrade your Formbricks deployment when using a local chart (e.g., with Minikube), use:
+
+```bash
+# From the helm-chart directory
+helm upgrade my-formbricks ./ --namespace formbricks
+```
+
+For installations from the Helm repository (typically for production deployments):
+```bash
+helm repo update
+helm upgrade my-formbricks formbricks/formbricks --namespace formbricks
+```
+
+### Common Upgrade Scenarios
+
+#### 1. Updating Environment Variables
+
+To update or add new environment variables, use the `--set` flag with the `env` prefix:
+
+```bash
+helm upgrade my-formbricks formbricks/formbricks \
+  --set env.SMTP_HOST=new-smtp.example.com \
+  --set env.SMTP_PORT=587 \
+  --set env.NEW_CUSTOM_VAR=newvalue
+```
+
+This command updates the SMTP host and port, and adds a new custom environment variable.
+
+#### 2. Enabling or Disabling Features
+
+You can enable or disable features by updating their respective values:
+
+```bash
+# Disable Redis
+helm upgrade my-formbricks formbricks/formbricks --set redis.enabled=false
+
+# Enable Redis
+helm upgrade my-formbricks formbricks/formbricks --set redis.enabled=true
+```
+
+#### 3. Scaling Resources
+
+To adjust resource allocation:
+
+```bash
+helm upgrade my-formbricks formbricks/formbricks \
+  --set resources.limits.cpu=1 \
+  --set resources.limits.memory=2Gi \
+  --set resources.requests.cpu=500m \
+  --set resources.requests.memory=1Gi
+```
+
+#### 4. Updating Autoscaling Configuration
+
+To modify autoscaling settings:
+
+```bash
+helm upgrade my-formbricks formbricks/formbricks \
+  --set autoscaling.minReplicas=3 \
+  --set autoscaling.maxReplicas=10 \
+  --set autoscaling.metrics[0].resource.target.averageUtilization=75
+```
+
+#### 5. Changing Database Credentials
+
+To update PostgreSQL database credentials:
+To switch from the built-in PostgreSQL to an external database or update the external database credentials:
+
+```bash
+helm upgrade my-formbricks formbricks/formbricks \
+  --set postgresql.enabled=false \
+  --set postgresql.externalUrl="postgresql://newuser:newpassword@external-postgres-host:5432/newdatabase"
+```
+
+This command disables the built-in PostgreSQL and configures Formbricks to use an external PostgreSQL database. Make sure your external database is set up and accessible before making this change.
+
+
+### Usage Examples
+
+Here are various examples of how to install and configure the Formbricks Helm chart:
+
+1. **Default Installation with Traefik enabled and Custom Hostname**:
```

Comment:
@Dhruwang I though the SSL certification wasn't working. Were you able to get it running in your cluster?